### PR TITLE
Implement sever side rendering for SEO

### PR DESCRIPTION
**Pull request by jorilindell**

### CODE REVIEW

File: client/.env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+REACT_APP_GRAPHQL_BASE_URL=http://localhost:4000/proxy/graphql
```
Comment (badrange) [inline client/.env]: These ones could go to `.env.development` (and that move can of course wait until we have a "production" environment)

**Comment by badrange** [comment]
Looks fine to me. I have so little experience with SSR that my opinion carries little weight. 

One thing that would be nice is more instructions on how to do debugging, especially of the SSR server.

File: client/config/env.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
+const fs = require("fs");
```
[the remark this answers is not in the document]
Comment (jorilindell) [reply client/config/env.js]: Yeah, these webpack related files are more or less same that create-react-app uses under the hood, but there is now way to re-use them without ejecting. These are for SSR to build the project

File: client/.env (hunk 1 above)
Comment (jorilindell) [reply client/.env]: I think that I'll wait to the point when we are talking about deployment

**Pull request merged**